x-pack/plugins/triggers_actions_ui/public/application/components/text_area_with_message_variables.tsx

```diff
@@ -48,6 +54,7 @@ export const TextAreaWithMessageVariables: React.FunctionComponent<Props> = ({
 
   const onChangeWithMessageVariable = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
     editAction(paramsProperty, e.target.value, index);
+    setWarning(validateParamsForWarnings(e.target.value, publicBaseUrl, messageVariables));
```

Comment:
I think you also need to add this check into `onSelectMessageVariable`. When I select `{{rule.url}}` from the dropdown, it does not trigger the check but when i edit the text manually, it does

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/149832/commits/64dcd1f24cc148fc1fb0148ad51da2e6e95dd392